packaging/helm-charts/helm3/strimzi-kafka-operator/crds/048-Crd-kafkamirrormaker2.yaml

```diff
@@ -244,6 +244,13 @@ spec:
                             type: integer
                             minimum: 1
                             description: The maximum number of tasks for the Kafka Connector.
+                          autoRestartConnectorsAndTasks:
```

Comment:
I guess the CRDs were not regenerated after you changed the field name to `autoRestart`?